src/Stepper/Stepper.jsx

```diff
@@ -0,0 +1,129 @@
+import React, {PropTypes} from 'react';
+
+import Mixins from '../mixins';
+import {getMuiTheme} from '../styles';
+
+const Stepper = React.createClass({
+  propTypes: {
+    /**
+     * The current active step index which passed by parent component
+     */
+    activeStepIndex: PropTypes.number,
+    /**
+     * Children should be Step type
+     */
+    children: PropTypes.node,
+    /**
+     * The icon will show when the step is completed
+     */
+    completedIcon: PropTypes.node,
```

Comment:
i think that's a right and cute way approach to customize the icons for every status of steps